config/manager/manager.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: etcd-cluster-operator
+  name: manager
```

Comment:
this may break the e2e tests as i think they rely on the operator being predictably named

Reply:
You were right. Fixed them now.